src/modules/pmc/src/lib/Pmc.h

```diff
@@ -2,10 +2,13 @@
 // Licensed under the MIT License.
 
 #include <PmcBase.h>
+#include <string>
 
```

Comment:
You can also create a common header for all includes this module needs and include that everywhere.